include/deal.II/base/vectorization.h

```diff
@@ -521,6 +521,29 @@ inline DEAL_II_ALWAYS_INLINE VectorizedArray<Number, width>
 
 
 
+/**
+ * Create a vectorized array that sets all entries in the array to the given
+ * scalar.
+ *
+ *  @relatesalso VectorizedArray
+ */
+template <typename VectorizedArrayType>
+inline DEAL_II_ALWAYS_INLINE VectorizedArrayType
+                             make_vectorized_array(const typename VectorizedArrayType::value_type &u)
+{
+  static_assert(
+    std::is_same<VectorizedArrayType,
+                 VectorizedArray<typename VectorizedArrayType::value_type,
+                                 VectorizedArrayType::n_array_elements>>::value,
+    "VectorizedArrayType is not a VectorizedArray.");
```

Comment:
Can you reflect this assertion in the documentation of the function?

Reply:
I will update the documentation.